backend/scheduler/tasks.py

```diff
@@ -49,15 +50,26 @@ def create_periodic_task(
         day_of_month=day_of_month,
         month_of_year=month_of_year,
     )
-    logger.info(f"Creating periodic task {task_name} with cron string: {cron_string}")
-
-    # Create periodic task
-    PeriodicTask.objects.create(
-        crontab=schedule,
-        name=task_name,
-        task=task_path,
-        args=task_args_json,
-    )
+    periodic_task = get_periodic_task(task_name=task_name)
+    if periodic_task:
+        periodic_task.crontab = schedule
+        periodic_task.enabled = enabled
+        periodic_task.args = task_args_json
+        periodic_task.save()
+        logger.info(
+            f"Updated periodic task {task_name} with cron string: {cron_string}"
+        )
+    else:
+        PeriodicTask.objects.create(
+            name=task_name,
+            task=task_path,
+            crontab=schedule,
+            enabled=enabled,
+            args=task_args_json,
+        )
+        logger.info(
+            f"Created periodic task {task_name} with cron string: {cron_string}"
+        )
```

Comment:
@muhammad-ali-e did you consider using [the update_or_create](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#update-or-create) here?

Reply:
Let't me check. I used some methods, But it not works. Thanks for the link.

Reply:
Updated